packages/GA4Client/test/src/tests.js

```diff
@@ -1522,6 +1522,7 @@ describe('Google Analytics 4 Event', function () {
                     {
                         page_title: 'Mocha Tests',
                         page_location: location.href,
+                        page_referrer: document.referrer,
```

Comment:
I see that you added some tests that include `page_referrer` but are there any tests cases that reflect the behavior when it isn't included?

Reply:
so as mentioned the default for page_referrer per google docs is document.referrer and I included it just how we default page_title and page_location